gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Policy.java

```diff
@@ -42,120 +40,63 @@
  *
  * @see <a href="https://cloud.google.com/iam/reference/rest/v1/Policy">Policy</a>
  */
-public class Policy extends IamPolicy<Policy.Role> {
+public class Policy extends IamPolicy<String> {
 
   private static final long serialVersionUID = -5573557282693961850L;
 
   /**
-   * Represents legacy roles in an IAM Policy.
+   * The project-level roles in an IAM policy. This enum is not an exhaustive list of all roles
+   * you can use in an IAM policy. You can also use service-specific roles (e.g.
+   * <i>roles/pubsub.editor</i>).  See the <i>Supported Cloud Platform Services</i> page for links
```

Comment:
Do you think we should make it clear that it is a string - "roles/pubsub.editor"?